### DISCUSSION TITLE
config.yaml

### DISCUSSION THREAD

**Discussion by thinzaung** [rancher/rke2#3168] [Q&A]
I am new to RKE2 configuration. I have recently build RKE2 cluster in HA mode. 
However, I would like to customize my ETC backup schedule and backup directory. Is it still possible for me to add the options in config.yaml and restart rke2-server service will take new configurations? 
Or do I have to rebuild everything from the scratch?

Thanks you so much in advance!!

**Comment by brandond** [comment]
Yes, you can change the configuration and restart rke2 to apply those configuration changes. There are a few things that should not be changed once the cluster is up, but the etcd snapshot config is not one of those things.

**Reply by thinzaung** [reply]
Thank you. I will try out.

**Comment by kkmaslowski** [comment]
@brandond You wrote that some options are not safe to change on running cluster. I added cloud-provider-name: rancher-vsphere to my running cluster to configure vsphere csi. Is it safe to change cloud-provider on running cluster? I restarted servers, installed from apps vsphere cpi, vsphere csi and storage is working correctly.
